src/sdam/server.ts

```diff
@@ -335,14 +338,16 @@ export class Server extends TypedEventEmitter<ServerEvents> {
         session.pin(checkedOut);
```

Comment:
This is as far up as I can comment in this method.  Just above this comment (lines 328-340), we have logic to handle load balanced mode separately.  We don't need to consider the operation count for load balanced mode, because there is only ever one server to select when in load balanced mode.  the load balanced logic above does call `server.command` recursively, so in that recursive call, we will increment and decrement the operation count.  That's okay - it won't affect server selection.